src/node_api.cc

```diff
@@ -2162,7 +2162,7 @@ napi_status napi_instanceof(napi_env env,
 
   if (env->has_instance_available) {
     napi_value value, js_result, has_instance = nullptr;
-    napi_status status;
+    napi_status status = napi_generic_failure;
```

Comment:
Because then everything will come tumbling down if you compare the value without setting it first - on every platform, all the time.